index.html

```diff
@@ -4,7 +4,7 @@
         <body>
             <head>
```

Comment:
Good job so far but i've noticed the incorrect use of the tag. The tag in HTML is used to define the head portion of the document which contains information related to the document. The tag contains other head elements such as <title>, , , <style> etc.
please instead make use of the <main> tag for the content of your page.